apps/test/unit/templates/sectionProgress/progressTables/ProgressTableStudentListTest.jsx

```diff
@@ -31,26 +34,44 @@ const DEFAULT_PROPS = {
   needsGutter: false
 };
 
+const setUp = (overrideProps = {}) => {
+  const store = createStore(
+    combineReducers({
+      sectionProgress
+    }),
+    {
+      sectionProgress: {
+        showSectionProgressDetails: false
```

Comment:
similar to my previous comment, imo would be more consistent to make an `UnconnectedProgressTableStudentList` and pass this as a prop.